R/cdfPlot.R

```diff
@@ -24,117 +21,226 @@
-#' @param pvalCol Name of the p-value or FDR column (Default = "P.Value")
-#' @param pvalMax Limit the range of the main plot (Default = 0.10)
+#' @param plotType Plot type must be canvasXpress or ggplot (default = canvasXpress).
+#' @param pvalCol Name of the p-value or FDR column (default = "P.Value")
+#' @param pvalMax Limit the range of the main plot (default = 0.10)
 #' @param pThreshold Used to color points (default = 0.01)
-#' @param xlab X axis label (default is "LogIntensity column name "Rank")
-#' @param ylab Y axis label (default is p-value column name)
+#' @param xlab X axis label (default = "Rank")
+#' @param ylab Y axis label (default = p-value column name)
 #' @param title Plot title (Optional)
 #' @param insetTitle Title for the inset plot (Optional)
-#' @param symbolSize Size of symbols for up, no change, and down. Default = c(4, 1, 4);
-#'        Note: All three cannot be the same size. Decimal values are acceptable to help offset that
-#'        (e.g. 4, 4.1, 4.2).
-#' @param symbolShape Shape of the symbols for up, no change, and down; Default = c(21, 20, 21)
-#'        (20 = filled circle, 21 = fillable open circle); Note: The same symbol shape cannot
-#'        be selected for all three symbols.
+#' @param symbolSize Size of symbols for Significant and Not Significant points (default = c(2,1);
+#' @param symbolShape Shape of the symbols for Significant and Not Significant points (default = c("circle", "circle")
 #'        See \url{http://www.cookbook-r.com/Graphs/Shapes_and_line_types}
-#' @param symbolColor c(Up, NoChange, Down); default = c("black", "grey25", "grey0")
+#' @param symbolColor Color of symbols for Significant and Not Significant points (default = c("red3", "deepskyblue4"))
 #'        See \url{http://research.stowers-institute.org/efg/R/Color/Chart}
-#'        Note: Colors cannot be duplicated.
-#' @param symbolFill Set the fill color for the symbols. Note only symbols 21-25 are fillable.
-#'        This will have no effect on other symbols.
-#'        Default = c("red3", "grey25", "deepskyblue4")
-#'        Note: Colors cannot be duplicated.
-#' @param alpha Controls the transparency of the plotted points (0-1; Default =
-#'   0.5)
+#' @param transparency Controls the transparency of the plotted points. Value ranges between 0 -1 (default = 0.5)
 #' @param referenceLine Color for an horizontal line drawn at the p-threshold
-#'   (Default = NULL; NULL disables, set to desired color to enable)
-#' @param refLineThickness Set thickness of the reference line (Default = 1)
-#' @param legendPosition (Default = "right")
-#' @param viewportX (Default = 0.15)
-#' @param viewportY (Default = 0.85)
-#' @param viewportWidth (Default = 0.35)
-#' @param printPlot Specify printing the combined plot to the console/knitr
-#'   (Default = TRUE)
+#'   (default = NULL; NULL disables, set to desired color to enable)
+#' @param refLineThickness Set thickness of the reference line (default = 1)
+#' @param viewportX (default = 0.15)
```

Comment:
I did not understand from documentation what **viewportX|Y** is

Reply:
done. Added a better definition